### PR TITLE
Rewrite the solution for Understanding Workflow and History

### PR DESCRIPTION
**Pull request by rgaiacs**
The solution was too long.

Some Markdown improvements in the question body is included.

### CODE REVIEW

File: _episodes/05-history.md [hunk 1]
```diff
@@ -381,93 +381,42 @@ moving backward and forward in time becomes much easier.
...
-> >
-> > ~~~
-> > $ cat venus.txt #this will print the contents of venus.txt to the screen
-> > ~~~
-> > {: .bash}
-> > If we print venus.txt we will get answer 2.
-> >
+> > The answer is 2 because `git add venus.txt` was used only before add the line
+> > `Venus is too hot to be suitable as a base`.
+> > And the flag `-a` was not used with `git commit`.
```
Comment (iglpdc) [inline _episodes/05-history.md]: Could you add a short sentence explaining that the second change was lost? Something like: "The second change was lost: it was not included in the commit and then overwritten by the `git checkout`.

Comment (rgaiacs) [reply _episodes/05-history.md]: > Could you add a short sentence explaining that the second change was lost? 

Done.

File: _episodes/05-history.md [hunk 2]
```diff
@@ -415,8 +415,9 @@ moving backward and forward in time becomes much easier.
 > > ## Solution
 > >
 > > The answer is 2 because `git add venus.txt` was used only before add the line
-> > `Venus is too hot to be suitable as a base`.
-> > And the flag `-a` was not used with `git commit`.
+> > `Venus is too hot to be suitable as a base`
+> > which was lost when `git checkout` was executed.
+> > Use the flag `-a` with `git commit` would have prevent the lost.
```
Comment (iglpdc) [inline _episodes/05-history.md]: Just a small grammar fix:
"Using the flag ... would have prevented the lost"  or "Use the flag ... to prevent the lost".

**Comment by iglpdc** [comment]
Thanks a lot, @rgaiacs !

**Pull request merged**